requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy>=1.15.4
 scipy>=1.1.0
-matplotlib
+matplotlib<3.4
```

Comment:
You'll have to change the `pip-pre` builds, too, if you're trying to just work around the error for now. Those reqs live in `tools/*dependencies*.sh` and are more of a pain to deal with. But maybe we can merge this PR as is so that it's at least just the `pip-pre` builds that are left failing until we find a fix

Reply:
Ok, I'll take it out and leave it failing